source/mir/ion/ser/msgpack.d

```diff
@@ -166,11 +326,149 @@ version(Have_msgpack_d)
             putValue(symbol);
         }
 
-        ///
-        void putValue(Num)(const Num num)
-            if (isNumeric!Num && !is(Num == enum))
+        void putValue(const ubyte num)
```

Comment:
All non template method shouldn't mark arguments as head const: just `ubyte num` is better